pkg/operator/staticpod/staticpodcontroller/deployment_controller.go

```diff
@@ -187,7 +187,7 @@ func (c DeploymentController) sync() error {
 
 	requeue, syncErr := c.createDeploymentController(operatorSpec, operatorStatus, resourceVersion)
 	if requeue && syncErr == nil {
-		return fmt.Errorf("synthetic requeue request")
+		return fmt.Errorf("synthetic requeue request (err: %v)", syncErr)
```

Comment:
nit: `... request: %v`, no?

Reply:
well, it can be nil and then you dunno what that <nil> means ;-)